src/style.css

```diff
@@ -2,2 +1,82 @@
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Roboto, sans-serif;
```

Comment:
Consider adding fallback fonts for the 'Roboto' font-family to ensure text is displayed with a similar style font if 'Roboto' fails to load.